app/src/main/java/org/mozilla/focus/customtabs/CustomTabConfig.java

```diff
@@ -163,7 +163,7 @@ public static CustomTabConfig parseCustomTabIntent(final @NonNull Context contex
         // Share is part of the default menu, so it's simplest just to toggle it off as necessary instead
         // of creating a fake menu item here, hence we keep this as  aboolean for now:
         final boolean showShareMenuItem = intent.getBooleanExtra(CustomTabsIntent.EXTRA_DEFAULT_SHARE_MENU_ITEM,
-                // Other than Chrome we will show the share menu by default. Note that currently the
+                // Unlike than Chrome we will show the share menu by default. Note that currently the
```

Comment:
Not a native speaker.. but in #1885 @mcomella says it should be "Unlike Chrome". This version says: "Unlike than Chrome".

Reply:
Sorry, my bad.